helm/keda/README.md

```diff
@@ -157,8 +155,7 @@ Kubernetes: `>=v1.24.0-0`
 | prometheus.webhooks.serviceMonitor.jobLabel | string | `nil` |  |
 | prometheus.webhooks.serviceMonitor.podTargetLabels | list | `[]` |  |
 | prometheus.webhooks.serviceMonitor.port | string | `"metrics"` |  |
-| prometheus.webhooks.serviceMonitor.relabellings[0].replacement | string | `"keda"` |  |
-| prometheus.webhooks.serviceMonitor.relabellings[0].targetLabel | string | `"app"` |  |
+| prometheus.webhooks.serviceMonitor.relabellings | list | `[]` |  |
```

Comment:
Also we should have the verticalPodAutoscaler section

Reply:
Did you run 'helm-docs helm/keda' and schema-gen ?

Reply:
It's strange because I din't change this manually. It was done by vendir

Reply:
That's probably because I did not run helm-doc in keda-upstream

Reply:
> Did you run 'helm-docs helm/keda' and schema-gen ?

I did run `helm-docs` but forgot `schema-gen`

Reply:
Can you run it again? Because I get different output

Reply:
Seems better I think